scroll-animations-1/Overview.bs

```diff
@@ -181,6 +181,13 @@ spec: cssom-view-1; type: dfn;
 		<dt><dfn>root</dfn>
 		<dd>
 			Specifies to use the document viewport as the [=scroll container=].
+
+		<dt><dfn>self</dfn>
+		<dd>
+			Specifies to use the element’s own [=principal box=] as the [=scroll container=].
+			If the [=principal box=] is not a [=scroll container=], 
+			or if there is no [=scrollable overflow=], 
```

Comment:
I think this condition at least should already be covered by the definition for [Scroll Progress Timelines](https://drafts.csswg.org/scroll-animations-1/#scroll-timelines):

> If the 0% position and 100% position coincide (i.e. the scroll container has no overflow to scroll), the timeline is [inactive](https://drafts.csswg.org/web-animations-1/#inactive-timeline).

Reply:
I think you're right, and this line should be removed.

```suggestion
```